### PR TITLE
Add Blogs and FAQs

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

File: mentorship/mentor/models.py [hunk 1]
```diff
@@ -210,6 +216,58 @@ def parent(self):
 ]
 
 
+class FAQIndexPage(Page):
+    parent_page_types = []
+    subpage_types = ['FAQ']
+
+
+class FAQ(Page):
```
Comment (miltontony) [inline mentorship/mentor/models.py]: Don't forget the `verbose_name` and `verbose_name_plural`

**Comment by saeedmb8** [comment]
@miltontony @justinvdm ready for review :)

File: mentorship/mentor/migrations/0024_data_migration_blogs_and_faqs_indexpage.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+from django.db import migrations, models
+
+
+def create_faqs_index(apps, schema_editor):
+    from mentor.models import FAQIndexPage, Main
+    main = Main.objects.all().first()
+
+    if main:
+        faq_index = FAQIndexPage(title='FAQ', slug='faq')
```
Comment (miltontony) [inline mentorship/mentor/migrations/0024_data_migration_blogs_and_faqs_indexpage.py]: Shouldn't this be `FAQs`?

File: mentorship/mentor/migrations/0024_data_migration_blogs_and_faqs_indexpage.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        faq_index.save_revision().publish()
+
+
+def create_blogs_index(apps, schema_editor):
+    from mentor.models import BlogIndexPage, Main
+    main = Main.objects.all().first()
+
+    if main:
+        blog_index = BlogIndexPage(
+            title='Blog', slug='blog')
```
Comment (miltontony) [inline mentorship/mentor/migrations/0024_data_migration_blogs_and_faqs_indexpage.py]: same here: `Blogs`

**Comment by miltontony** [comment]
Some small comments, otherwise 👍

**Pull request merged**